packages/cli/src/tools/config/errors.ts

```diff
@@ -31,5 +31,9 @@ export class JoiError extends CLIError {
         })
         .join(),
     );
+
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, JoiError);
+    }
```

Comment:
Getting rid of internal stack, pointing to a place where the error was thrown

<img width="839" alt="image" src="https://user-images.githubusercontent.com/5106466/83192064-4ddeb100-a135-11ea-94ca-f6e45eb5e963.png">